src/Function/Bundles.agda

```diff
@@ -526,6 +526,16 @@ module _ {A : Set a} {B : Set b} where
     , strictlyInverseʳ⇒inverseʳ to invʳ
     )
 
+------------------------------------------------------------------------
+-- Inline bbreviations for oft-used items
+--  (but one define the ones that recur)
+------------------------------------------------------------------------
+
+-- Same convention as above, with appended ₛ (for 'S'etoid)
```

Comment:
Very minor but it would make more sense to me for those to live the above the propositional definitions, as they're more general.